app/general/models.py

```diff
@@ -84,36 +101,49 @@ class DocumentFile(models.Model):
                     More information about Creative Commons licenses.
```

Comment:
This help text is not marked. Let us mark just this part ("More information about Creative Commons licenses."). That way translators don't need to bother with the surrounding HTML.

Reply:
will fix the above